solr_plugins/Bibelstellensuche/src/de/uni_tuebingen/ub/ixTheo/bibleRangeSearch/BibleRange.java

```diff
@@ -0,0 +1,37 @@
+package de.uni_tuebingen.ub.ixTheo.bibleRangeSearch;
+
+public class BibleRange extends Range {
+
+	public BibleRange(final String range) {
+		super(getLower(range), getUpper(range));
+	}
+
+	private static int getLower(String range) {
+		if (range.length() == 15) {
+			return Integer.valueOf(range.substring(0, 7));
+		} else {
+			return 0;
+		}
+	}
+
+	private static int getUpper(String range) {
+		if (range.length() == 15) {
+			return Integer.valueOf(range.substring(9, 15));
+		} else {
+			return 9999999;
```

Comment:
Don't we also have the seven nines in the string or do we really need to special-case this?